website/templates/website/honeycomb.html

```diff
@@ -18,6 +18,16 @@
     </div>
     {% endfor %}  
   </div>
+  <div class="youtube-gallery">
+    <h2>Youtube Gallery</h2>
+    {% for video in all_youtube_videos %}
+        {% if video.id.kind == "youtube#video" %}
```

Comment:
What other types?
For videos I guess everything will be in youtube channels. And for the tutorials I am still thinking what to do. I can provide a list view like a previous site. But may be if somehow if we could extract the images associated with each tutorials then we could make a gallery of thumbnails.